src/mindlessgen/cli/entrypoint.py

```diff
@@ -45,9 +45,6 @@ def console_entry_point(argv: Sequence[str] | None = None) -> int:
     if molecules:
         for molecule in molecules:
             try:
-                molecule.write_xyz_to_file()
```

Comment:
What is the purpose of changing the location of the print statement?

Reply:
If the print statement is located in `entrypoint.py`, the `xyz` files are only written to disk after all `num_molecules` (see `GeneralConfig` for this) molecules were generated. If one wants to have a look at the structures during the generation process, we have to shift the file writing in the actual `generation` function.